alacritty/src/event.rs

```diff
@@ -675,6 +654,44 @@ impl<N: Notify + OnResize> Processor<N> {
         }
     }
 
+    pub fn reload_config<T>(
+        path: &PathBuf,
+        processor: &mut input::Processor<T, ActionContext<N, T>>,
+    ) where
+        T: EventListener,
+    {
+        processor.ctx.message_buffer.remove_target(LOG_TARGET_CONFIG);
+        processor.ctx.display_update_pending.message_buffer = Some(());
+
+        if let Ok(config) = config::reload_from(&path) {
```

Comment:
```suggestion
        if let Ok(config) = config::reload_from(&path) {
        let config = match config::reload(&path) {
            Ok(config) => config,
            Err(_) => return, // Maybe log to debug?
        }
```

Reply:
This is already logged upper.

Reply:
Also, clippy is against your change(asks for `if let`), especially you take into account that we log in `config::reload` already, since it's used from multiple paths.

Reply:
As discussed in IRC, this should still be done.

Reply:
was done.